app/view_models/auction_show_view_model.rb

```diff
@@ -140,6 +140,14 @@ def lowest_bidder_name
     auction.lowest_bid.bidder.name
   end
 
+  def admin_edit_auction_partial
+    if UserPresenter.new(current_user).admin?
```

Comment:
thoughts on having this be `current_user.decorate.admin_edit_auction_partial` and then having a `admin_edit_auction_partial` method in `UserPresenter` and `GuestPresenter` ?